cmd/launcher/main.go

```diff
@@ -34,14 +36,27 @@ func runLaunch() error {
 		return cmd.FailErr(err, "read metadata")
 	}
 
+	var ex launch.ExecFunc
+	if runtime.GOOS == "windows" {
+		ex = func(argv0 string, argv []string, envv []string) error {
+			c := exec.Command(argv0, argv...)
+			c.Stdout = os.Stdout
+			c.Stderr = os.Stderr
+			c.Env = append(os.Environ(), envv...)
```

Comment:
Is there any danger in setting `os.Environ()` directly on the command? Should it not be `env.NewLaunchEnv(os.Environ())`? I see that we're passing `l.Env.List()` when we call `Launch`, which is appending to what's already there - but is this what we want? (I am just coming up to speed on the luancher, so apologies if this is an erroneous question...)

Reply:
Thanks for pointing this out. I'll look into it.

Reply:
@natalieparellano After looking at this, I think appending to `os.Environ()` here is extraneous, and we can just set `c.Env = envv`. As you mentioned, we're passing `l.Env.List()` in as `envv`, so it should already have `os.Environ()` in it.

Reply:
After reading through `NewLaunchEnv` again I agree, and in fact I think we explicitly do not want to pass `os.Environ()` by itself - since variables that should be excluded are stripped out in that call: https://github.com/buildpacks/lifecycle/blob/9e53a47b65ff169c0c96fd518f006e23fcca25d7/env/launch.go#L12